plugins/hooks/mattermost_webhook_hook.py

```diff
@@ -0,0 +1,83 @@
+from airflow.plugins_manager import AirflowPlugin
+import json
+from airflow import AirflowException
+# from airflow.providers.http.hooks.http import HttpHook -> airflow 2
+from airflow.hooks.http_hook import HttpHook
+
+
+class MattermostWebhookHook(HttpHook):
+
+    """
+    Send messages to Mattermost using webhooks. For more details on the parameters, please check
+    `Incoming webhooks <https://developers.mattermost.com/integrate/webhooks/incoming/#parameters>_` documentation.
+
+    :param http_conn_id: :ref:`http connection<howto/connection:http>` that has the base
```

Comment:
```suggestion
    :param mattermost_conn_id: :ref:`http connection<howto/connection:http>` that has the base
```

Small inconsistency between docs and code.